### ISSUE TITLE
dpkg-build fails on debian

### ISSUE THREAD

**Issue by Mathias-Fuchs** [mosra/corrade#44]
Hello dear authors ... this is what I get trying to build on debian:

```
mathias@lucky3:~/corrade$ dpkg-buildpackage
dpkg-buildpackage: info: source package corrade
dpkg-buildpackage: info: source version 2018.02-1
dpkg-buildpackage: info: source distribution unstable
dpkg-buildpackage: info: source changed by Vladimír Vondruš <mosra@centrum.cz>
dpkg-buildpackage: info: host architecture amd64
 dpkg-source --before-build corrade
 fakeroot debian/rules clean
dh clean --parallel
   dh_testdir -O--parallel
   dh_auto_clean -O--parallel
   dh_clean -O--parallel
 dpkg-source -b corrade
dpkg-source: error: can't build with source format '3.0 (native)': native package version may not have a revision
dpkg-buildpackage: error: dpkg-source -b corrade gave error exit status 255
```
....

any help appreciated.

**Comment by mosra** [comment]
Hi, thanks for bringing this up! :)

This was just a warning on Ubuntu, so I didn't really care, was not aware it's a hard error on Debian. Should be fixed in 5d26fe64a9250b87d2ebc7d7240844adbfb37337, I'm now going to fix it for all other projects as well. Please reopen if the issue persists.

**Comment by Mathias-Fuchs** [comment]
ok thanks!
